packages/connect-wallet-modal/src/components/Ledger/LedgerAccountScreen.tsx

```diff
@@ -70,7 +70,7 @@ export const LedgerAccountScreen: FC<Props> = ({ metrics, closeScreen }) => {
         await connect();
         closeScreen?.();
       } catch (e) {
-        setError(interceptLedgerError(e as Error));
+        setError(helpers.interceptLedgerError(e as Error));
```

Comment:
Shouldn't we change if it's actually an error and show something else if it's not?

Reply:
Or we can change signature to support unknown if it doesn't really matter
```
export const interceptLedgerError = (error: unknown) => {
```

Reply:
It is always Error here, and it must be always Error here. I don't know what else can be shown if it is suddenly not Error.

Reply:
At least setting `interceptLedgerError` argument type to unknown will save you from casting `e as Error` everywhere, but it's up to you